schemas/context/person.example.1.json

```diff
@@ -5,5 +5,10 @@
     "xdm:lastName": "Doe",
     "xdm:fullName": "Jane F. Doe"
   },
-  "xdm:birthDate": "1996-01-03"
+  "xdm:birthDate": "1996-01-03",
```

Comment:
According to the documentation, you should *not* have `birthYear` or `birthDayAndMonth` if you know the full date.   Please remove.

Reply:
Those values are straight from the existing schema.  Why would we not put them in the example?

Reply:
Read the documentation the schema - it is very clear that you *do not* create an actual JSON packet like this.  If you want, create multiple examples, each correct with different sets of values.

Reply:
Resolved in latest commit